plugins/modules/keycloak_group.py

```diff
@@ -69,6 +71,33 @@
             - A dict of key/value pairs to set as custom attributes for the group.
             - Values may be single values (e.g. a string) or a list of strings.
 
+    parents:
+        version_added: "6.3.0"
+        type: list
+        description:
+            - List of parent groups for the group to handle sorted top to bottom.
+            - Set this to create a group as a subgroup of another group or groups (parents).
```

Comment:
Is this only used during creation, or also during updating?

Reply:
I added some more docu here to basically explicitly list all cases when and when not using this parameter is necessary. What do you think, does it do the trick now?

Reply:
I think it sounds good! Thanks.